tools/help.txt

```diff
@@ -602,18 +603,23 @@ information, builds the package and sends both the package source and the built
 version of the package to the package server.
 
 This will only create one build of the package. If your package has multiple
-builds for different OS architectures, it will create a build for this machine's
-architecture. To publish a different build for the same version, run
-publish-for-arch.
+builds for different OS architectures, it will ask you to use the Meteor build
+farm to create a build per each supported architecture. To publish a different
```

Comment:
I think "for" is better than "each" here.

Also this isn't actually true any more now that publish never creates builds for binary packages, right? This should be more like:  "If the package has an OS-dependent binary component, this command publishes metadata about the package but does not actually publish builds for the package; use publish-for-arch probably via get-machine"